test/nimble_template/addons/github_test.exs

```diff
@@ -123,4 +123,132 @@ defmodule NimbleTemplate.Addons.GithubTest do
       end)
     end
   end
+
+  describe "#apply/2 with github_wiki option" do
+    test "copies the .github/workflows/publish_wiki.yml and Github Wiki files", %{
+      project: project,
+      test_project_path: test_project_path
+    } do
+      in_test_project(test_project_path, fn ->
+        Addons.Github.apply(project, %{github_wiki: true})
+
+        assert_file(".github/workflows/publish_wiki.yml")
+
+        assert_file(".github/wiki/Getting-Started.md", fn file ->
+          assert file =~ "Erlang 24.0.4"
+          assert file =~ "Elixir 1.12.2"
+
+          assert file =~ """
+                 - Install Node dependencies:
+
+                   ```sh
+                   npm install --prefix assets
+                   ```
+                 """
+
+          assert file =~ """
+                 - Start the Phoenix app
+
+                   ```sh
+                   iex -S mix phx.server
+                   ```
+                 """
+        end)
+
+        assert_file(".github/wiki/Home.md", fn file ->
+          assert file =~ "Insert information about your project here!"
+        end)
+
+        assert_file(".github/wiki/_Sidebar.md", fn file ->
+          assert file =~ "Table of Contents"
+        end)
+      end)
+    end
+  end
+
+  describe "#apply/2 with api_project and github_wiki option" do
+    test "copies the .github/workflows/publish_wiki.yml and Github Wiki files", %{
+      project: project,
+      test_project_path: test_project_path
+    } do
+      in_test_project(test_project_path, fn ->
```

Comment:
Do we need to do:  
```
project = %{project | api_project?: true, web_project?: false}
```

Maybe later, we might want to create a helper like `@describetag mix_project?: true` to setup api project for testing to make it more convenient. 🤔 💡